es4x/src/main/java/io/reactiverse/es4x/Runtime.java

```diff
@@ -33,13 +33,13 @@
 import java.util.Map;
 import java.util.function.Function;
 
-public final class Runtime extends EventEmitterImpl {
+public class Runtime extends EventEmitterImpl {
 
-  private final Context context;
-  private final Value bindings;
-  private final Value module;
+  protected final Context context;
+  protected final Value bindings;
+  protected final Value module;
 
-  private static final Source[] POLYFILLS = new Source[] {
+  protected static final Source[] POLYFILLS = new Source[] {
```

Comment:
This is dangerous as it requires change of a static field before a new instantiate happens. This can lead to undefined behavior.

Reply:
My intent was not to change content of this array just happen that I was required to access it  when creating a custom runtime. a simple copy will do the job but also if there is some way to reuse without copying code it would be better.